src/groundlight/client.py

```diff
@@ -123,37 +125,24 @@ def submit_image_query(
         self,
         detector: Union[Detector, str],
         image: Union[str, bytes, BytesIO, BufferedReader, np.ndarray],
```

Comment:
nit: add `Image.Image` to the supported input types

Reply:
Good catch!